assets/stylesheets/bootstrap.scss

```diff
@@ -5,3 +5,4 @@
 @import 'base/modules';
 @import 'components/modules';
 @import 'themes/modules';
+@import 'utilities/modules';
```

Comment:
Does it make more sense to do this and refactor @zarahzachz's newly added helper classes into `utilities/modules` or move these classes into her file? 

https://github.com/crdschurch/crds-styles/blob/development/assets/stylesheets/base/_utilities.scss